packages/block-editor/src/components/iframe/index.js

```diff
@@ -28,68 +27,85 @@ const BODY_CLASS_NAME = 'editor-styles-wrapper';
 const BLOCK_PREFIX = 'wp-block';
 
 /**
- * Clones stylesheets targetting the editor canvas to the given document. A
- * stylesheet is considered targetting the editor a canvas if it contains the
- * `editor-styles-wrapper`, `wp-block`, or `wp-block-*` class selectors.
+ * Hook that clones stylesheets targetting the editor canvas to the given
+ * document. A stylesheet is considered targetting the editor a canvas if it
+ * contains the `editor-styles-wrapper`, `wp-block`, or `wp-block-*` class
+ * selectors.
  *
  * Ideally, this hook should be removed in the future and styles should be added
  * explicitly as editor styles.
- *
- * @param {Document} doc The document to append cloned stylesheets to.
  */
-function styleSheetsCompat( doc ) {
-	// Search the document for stylesheets targetting the editor canvas.
-	Array.from( document.styleSheets ).forEach( ( styleSheet ) => {
-		try {
-			// May fail for external styles.
-			// eslint-disable-next-line no-unused-expressions
-			styleSheet.cssRules;
-		} catch ( e ) {
-			return;
-		}
-
-		const { ownerNode, cssRules } = styleSheet;
+const useEditorStyles = () =>
```

Comment:
Minor: it's better to use named functions for debugging.

Reply:
Can we also document that this is a compatibility layer, if we remove `compat` from the name?

Reply:
Thank you for reviewing! Updated in e61ecabdb3fbab1648ffde0938334e4577903074

Reply:
I missed another arrow function, fixed in e752a13098cbbf540022365dbed217b776971781